src/js/nav/sourceOfTruth.js

```diff
@@ -19,6 +19,11 @@ module.exports = (cachePrefix) => {
 
     instance.interceptors.response.use((response) => response.data || response);
 
-    return instance.get('https://raw.githubusercontent.com/'
-    + 'RedHatInsights/cloud-services-config/master/main.yml');
+    // TODO: Add prefix (/beta) depending on environment
+    let prefix = '';
+    if (window.location.pathname.indexOf('/beta')) {
```

Comment:
This logic needs to flip

Reply:
It needs update actually
`if (window.location.pathname.indexOf('/beta') !== -1) {`

Reply:
Yeah, I'd probably roll with that pattern since we're using that in other places too.